### DISCUSSION TITLE
Is this the only tool that installs and manages python versions

### DISCUSSION THREAD

**Discussion by ciscorucinski** [pyenv/pyenv#3026] [General]
After getting used to Android Studio's SDK Manager, I had difficulty finding similar ease-of-use tools for other SDKs like Python to install new versions and updates simply. 

I have pyenv running on my machine now, but I am curious if there are alternatives. 

I run Linux. I am okay with using a cli, but gui suggestions are good, too. I don't want to use the python website to download and figure out where to correctly install each new python version so that it doesn't override my system python.

**Comment by tomschr** [comment]
@ciscorucinski Maybe have a look at [Rye](https://rye.astral.sh)? It uses the [indygreg](https://github.com/indygreg/python-build-standalone) standalone builds.

**Reply by ciscorucinski** [reply]
Thanks, I had missed, or not understood, `rye toolchain ...`. I'll look into this more!
